docs/release_notes/release_notes.md

```diff
@@ -11,8 +11,67 @@ has_children: true
 
 ## Version 4.9.0 (July 2024)
 
+* Added support for Nessus:Pro (Nessus Professional). Check the upgrade guide to onboard the logs.
+
+* Added checkbox on alert page to prevent the alert state from being updated from the Cyences setup page. Check the upgrade guide for more details.
+
+* Added the following panels for user login activity on the **Windows** dashboard:
+    * Successful Login Events
+    * Failed Login Attempts by User
+
+* Added the following panels to the **Vulnerability** dahboard:
+    * Total Vulnerability Count By Severity
+    * New Total Vulnerability Found Over Time
+
+* Added a separate alert for password expired events called **AD - Login Failure due to Password Expired** and filtered out the password expired events from authentication alerts.
+
+* Added a section to configure the digest email configs on **Cyences Settings > Cyences App Configuration > Cyences Email Action Configuration**.
+
+* Removed the following alerts as it contains static lookup which is not maintainable and causes false positives:
```

Comment:
Removed the following alerts as it contains static lookup causing many false positives.